Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1939,7 +1939,7 @@ open class NavigationMapView: UIView {
         mapView.ornaments.options.scaleBar.visibility = .hidden
         storeLocationProviderBeforeSimulation()
         
-        mapView.mapboxMap.onEvery(.renderFrameFinished) { [weak self] _ in
+        mapView.mapboxMap.onEvery(event: .renderFrameFinished) { [weak self] _ in
```

Comment:
Do we have breaking changes in Maps SDK?

Reply:
Not breaking change, but deprecation in this cases. Modified signature to fix warnings.

Reply:
Perfect, thanks